### PR TITLE
Remove unnecessary Advisory properties from Advisory models in Estuary

### PR DESCRIPTION
**Pull request by sarah256**
@mprahl could you please review this :)

### CODE REVIEW

File: scrapers/errata.py [hunk 1]
```diff
@@ -156,7 +140,6 @@ def get_advisories(self, since, until):
                 main.created_at,
                 main.id AS id,
                 main.issue_date,
-                package_users.login_name AS package_owner,
```
Comment (mprahl) [inline scrapers/errata.py]: Could you also remove the left join on `Errata_public.users AS package_users`?

File: scrapers/errata.py [hunk 2]
```diff
@@ -179,30 +161,12 @@ def get_advisories(self, since, until):
                 ON main.package_owner_id = package_users.id
             LEFT JOIN Errata_public.users AS reporter_users
                 ON main.reporter_id = reporter_users.id
-            WHERE main.updated_at >= '{0}' AND main.updated_at <= '{1}'
-            ORDER BY main.updated_at DESC;
+            WHERE main.updated_date >= '{0}' AND main.updated_date <= '{1}'
```
Comment (mprahl) [inline scrapers/errata.py]: This should be `update_date` and not `updated_date`

File: tests/api/test_general.py [hunk 3]
```diff
@@ -130,7 +122,7 @@ def test_get_on_model_wo_uid(client, resource):
     rv = client.get('/api/v1/{0}/some_repo'.format(resource))
     assert rv.status_code == 400
     invalid_msg = ('The requested resource "{0}" is invalid. Choose from the following: '
-                   'advisory, advisorystate, bugzillabug, containeradvisory, containerkojibuild, '
+                   'advisory, bugzillabug, containeradvisory, containerkojibuild, '
                    'distgitcommit, freshmakerevent, kojibuild, kojitag, kojitask, and user.'
```
Comment (mprahl) [inline tests/api/test_general.py]: Optional: You could move some of the start of this string to the end of the line above

File: tests/api/test_get_resource.py [hunk 4]
```diff
@@ -587,10 +560,8 @@ def test_get_resources(client, resource, uid, expected):
 
     if resource == 'advisory':
         advisory.assigned_to.connect(mprahl)
-        advisory.package_owner.connect(tbrady)
```
Comment (mprahl) [inline tests/api/test_get_resource.py]: So sad that you're removing a connection with Tom Brady :cry:

Comment (sarah256) [reply tests/api/test_get_resource.py]: Nick Foles is the new GOAT sorry :1st_place_medal:

File: scrapers/errata.py [hunk 5]
```diff
@@ -166,43 +149,22 @@ def get_advisories(self, since, until):
...
             LEFT JOIN Errata_public.users AS assigned_users
                 ON main.assigned_to_id = assigned_users.id
-            LEFT JOIN Errata_public.users AS package_users
-                ON main.package_owner_id = package_users.id
             LEFT JOIN Errata_public.users AS reporter_users
                 ON main.reporter_id = reporter_users.id
-            WHERE main.updated_at >= '{0}' AND main.updated_at <= '{1}'
-            ORDER BY main.updated_at DESC;
+            WHERE main.update_date >= '{0}' AND main.update_date <= '{1}'
+            ORDER BY main.updated_date DESC;
```
Comment (yashvardhannanavati) [inline scrapers/errata.py]: Could you please change this as well from `updated_date` to `update_date`.

File: tests/api/test_get_resource.py (hunk 4 above)
Comment (mprahl) [reply tests/api/test_get_resource.py]: Tom Brady is the :goat:!

**Comment by yashvardhannanavati** [comment]
:+1:

**Pull request merged**